visualizer/src/audio/util.hpp

```diff
@@ -63,7 +63,12 @@ namespace audio {
             audio::debug_info("[INFO: AUDIO]: FOUND THE SCHIIT BOX");
 
             /* SCHIIT BOX settings */
-            const PaStreamParameters output_parameters = {.channelCount = 2, .device = the_schiit_box, .sampleFormat = paFloat32};
+            const PaStreamParameters output_parameters = {
+                .device = the_schiit_box,
+                .channelCount = 2,
+                .sampleFormat = paFloat32,
+                .suggestedLatency = PaTime(5.0),
+                .hostApiSpecificStreamInfo = nullptr};
```

Comment:
g++ on ubuntu errors on non-strict import order for compiler extension initialization syntax